Sources/AddressURL/IPv4Address.swift

```diff
@@ -20,3 +20,19 @@ extension IPv4Address {
         return URL(address: self)
     }
 }
+
+public struct IPv4AddressWithPort {
```

Comment:
I don't see a point for this new structure.

Reply:
Acknowledged. Did remove below.